gateway/modules/crud/sql/read.go

```diff
@@ -109,6 +132,17 @@ func (s *SQL) generateReadQuery(col string, req *model.ReadRequest) (string, []i
 	}
 	return sqlString, args, nil
 }
+func generateAggregateAsColumnName(function, column string) string {
+	return fmt.Sprintf("%s__%s__%s", utils.GraphQLAggregate, function, column)
+}
+
+func splitAggregateAsColumnName(asColumnName string) (functionName string, columnName string, isAggregateColumn bool) {
+	v := strings.Split(asColumnName, "__")
+	if len(v) != 3 {
```

Comment:
This is wrong if its an aggregate column then the length will be 3 and it will always start from prefix aggregate.
because of this logic, I won't get result in output

Reply:
What. If the user has a similar naming convention? Also, i forgot the `!`